File: CS/Demo.Module/DatabaseUpdate/Updater.cs
using System;
using Demo.Module.BusinessObjects;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;

namespace Demo.Module.DatabaseUpdate {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            if(ObjectSpace.GetObjectsCount(typeof(Contact), null) == 0) {
                for(int i = 0; i < 10; i++) {
                    DatabaseHelper.CreateContact(ObjectSpace);
                    DatabaseHelper.CreateAddress(ObjectSpace);
                    DatabaseHelper.CreateDocument(ObjectSpace);
                    this.UpdateStatus("Creating test data", "", string.Format("Batch #{0} was created", i));
                }
            }
            ObjectSpace.CommitChanges();
        }
    }
}

File: VB/Demo.Module/DatabaseUpdate/Updater.vb
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating

Namespace Demo.Module.DatabaseUpdate

    Public Class Updater
        Inherits ModuleUpdater

        Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
            MyBase.New(objectSpace, currentDBVersion)
        End Sub

        Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
            MyBase.UpdateDatabaseAfterUpdateSchema()
            For i As Integer = 0 To 5 - 1
                CreateContact(ObjectSpace)
                CreateAddress(ObjectSpace)
                CreateDocument(ObjectSpace)
                UpdateStatus("Creating test data", "", String.Format("Batch #{0} was created", i))
            Next

            ObjectSpace.CommitChanges()
        End Sub
    End Class
End Namespace
